recipes/windows_reboot.rb

```diff
@@ -4,9 +4,19 @@
 
 include_recipe 'windows::reboot_handler'
 
-windows_reboot 'rebooter' do
+windows_reboot 'powershell' do
   reason 'Reboot after successful/unsuccessful powershell installation'
-  timeout 60
-  notifies :run, node['powershell']['reboot_notifier'], :immediately
-  not_if { false }
+  timeout 10
```

Comment:
The timeout is how many seconds from the shutdown command being run (when chef finishes) to when the machine actually reboots. I'm not sure there is much else to happen in that time... Maybe finish the report handler? 

The nice thing about doing it quickly is that processes that need to reconnect after a reboot have a smaller window where they can log in just prior to the machine rebooting. 

I'll change it to use an attribute so it is at least easily configurable.